test/fuse-test.sh

```diff
@@ -453,6 +453,40 @@ function test_write_after_seek_ahead {
    rm testfile
 }
 
+function test_rmdir_withfdopen {
+    
+    mkdir -p  $TEST_BUCKET_MOUNT_POINT_1/$TEST_DIR/test
+    mkdir -p  $TEST_BUCKET_MOUNT_POINT_1/$TEST_DIR/test1
+    echo "ls mountpoint"
+    lsout0=`ls -lah $TEST_BUCKET_MOUNT_POINT_1/`
+    echo $lsout0
+
+
+
+   nohup /root/go/src/github.com/kahing/goofys/test/rmdirwihtopen/lsunconsistency $TEST_BUCKET_MOUNT_POINT_1/$TEST_DIR/test a > /dev/null 2>&1 &
```

Comment:
what is this lsunconsistency?

Reply:
ah found that you committed a binary. Please replace that with the source (maybe C) file instead. I am not going to run a random binary to verify the problem